plugins/cilium-cni/types/types.go

```diff
@@ -30,9 +30,10 @@ import (
 // NetConf is the Cilium specific CNI network configuration
 type NetConf struct {
 	cniTypes.NetConf
-	MTU  int              `json:"mtu"`
-	Args Args             `json:"args"`
-	ENI  ciliumv2.ENISpec `json:"eni,omitempty"`
+	MTU          int              `json:"mtu"`
+	Args         Args             `json:"args"`
+	ENI          ciliumv2.ENISpec `json:"eni,omitempty"`
+	DisableDebug bool             `json:"disable-debug"`
```

Comment:
Any reason for not making this opt-in? Because we defaulted to debug enabled in the past?

Reply:
@tgraf correct

Reply:
That is clearly wrong behavior so I think we should disable it for all new versions unless explicitly disabled. I don't see a particular reason to continue providing debug messages by default.

Reply:
changed